src/enum_net.cpp

```diff
@@ -83,6 +83,11 @@ POSSIBILITY OF SUCH DAMAGE.
 #include "libtorrent/aux_/windows.hpp"
 #include <iphlpapi.h>
 #include <ifdef.h> // for IF_OPER_STATUS
+#ifdef TORRENT_WINRT
+using NDIS_MEDIUM = int;
+using NDIS_PHYSICAL_MEDIUM = int;
```

Comment:
if these don't exist, does it work to enumerate routes and adapters at all?

Reply:
Surprisingly it does work, on Desktop and Mobile; the `ret` vector in `enum_net_interfaces` got filled with something reasonable-looking. (it even found cellular devices!)

Reply:
I get the impression that the existence of these types is related to the version of the SDK, perhaps in addition to whether it's a WinRT build.

Reply:
Yep, might actually be a bugfix in a newer version, I'm using 17763.